docs/RELEASE.md

```diff
@@ -160,9 +160,12 @@ Describes how developers should merge bug and security fixes.
 
 ### Steps
 
-1. Developer opens a Pull Request with a bug or security fix.
-2. The Pull Request is marked as priority fix.
-3. Audit happens with priority.
+1. Developer opens a Pull Request into `master` with a bug or security fix.
+2. The Pull Request is marked as priority-fix.
+3. Developer notifies the auditors of its relevance.
 4. It is merged into `master`.
-5. It is automatically back-ported to `stable`.
-6. The fix will be released in the next *Stable* release. In urgent cases, a release can happen earlier.
+5. Developer opens a backport Pull Request into the `stable` branch.
+6. CI validates that there there are no obvious major SemVer change.
+7. Audit must be completed by now.
```

Comment:
I reworded it as i dont know what severity to put here, maybe the Security team can put something in.  
Otherwise the developer should be able to judge this.